### PR TITLE
created login component

### PR DESCRIPTION
**Pull request by TharunChamarthi**
created sample Login Page, as login Component without page responsiveness.

### CODE REVIEW

**Comment by TharunChamarthi** [comment]
created sample Login Page, as login Component without page responsiveness.
Screenshot 2024-07-03 at 3 38 11 AM

File: src/app/app.component.ts [hunk 1]
```diff
@@ -1,13 +1,29 @@
...
+    ReactiveFormsModule,
+    MatIconModule,
+  ],
   templateUrl: './app.component.html',
-  styleUrl: './app.component.scss'
+  styleUrl: './app.component.scss',
 })
 export class AppComponent {
-  title = 'temp-project';
+  title = 'Order.It';
```
Comment (suryakolli) [inline src/app/app.component.ts]: Keep it consistent, let us use name of the app as order.it with all small case letters. Also extract it out and store it in a separate constants file.

File: src/app/app.routes.ts [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+import { LoginComponent } from './login/login.component';
 
-export const routes: Routes = [];
+export const routes: Routes = [
+  {
+    path: '',
+    component: AppComponent,
+    children: [
+      {
+        path: '',
```
Comment (suryakolli) [inline src/app/app.routes.ts:12]: add a path such that only login component is called when /login route is called. and not for  home path

File: src/app/login/login.component.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<mat-card>
+  <mat-card-content>
+    <div class="header">
+      <h2>LOGIN</h2>
```
Comment (suryakolli) [inline src/app/login/login.component.html]: Make LOGIN to Login.

File: src/app/login/login.component.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    </div>
+    <form (ngSubmit)="(onLogin)" name="loginForm" [formGroup]="loginForm">
+      <div class="email-input">
+        <mat-form-field class="full-width" appearance="outline">
```
Comment (suryakolli) [inline src/app/login/login.component.html:8]: take care of the classes like full-width and make the form visually appealing

**Review by suryakolli — CHANGES_REQUESTED** [review]
image

Make the login compoennt centered to the page and look like this.

File: src/app/app.component.ts (hunk 1 above)
Comment (TharunChamarthi) [reply src/app/app.component.ts]: imported title form constants.ts file

File: src/app/login/login.component.html (hunk 3 above)
Comment (TharunChamarthi) [reply src/app/login/login.component.html]: updated header

**Review by suryakolli — APPROVED** [review]

**Pull request merged**